### DISCUSSION TITLE
Yattee gave me 4h+ more battery on M1 PRO.

### DISCUSSION THREAD

**Discussion by 33ddy** [yattee/yattee#306] [Ideas]
After many tries and monitoring my M1 Pro cpu and battery, I noticed playing youtube videos the normal way in the browser consume so much power I speak about 4h+ different.
So I directed the videos to open in Yattee using Choosy or extension+Bettertouchtool.
You can use Velja free alternative to Choosy.

The setting if someone want it.

<img width="1129" alt="Screenshot 2022-09-16 at 20 46 16" src="https://user-images.githubusercontent.com/51548107/190710416-0b455f43-6238-44f3-ba84-8a25a1f66df5.png">

<img width="602" alt="Screenshot 2022-09-17 at 19 13 41" src="https://user-images.githubusercontent.com/51548107/190868827-ce890d85-e99c-4b67-b634-8159cd892456.png">

Choosy Rules

<img width="619" alt="Screenshot 2022-09-17 at 19 15 01" src="https://user-images.githubusercontent.com/51548107/190868828-67845c15-1d94-4fbe-ba1c-21938a74d463.png">

<img width="619" alt="Screenshot 2022-09-17 at 19 15 51" src="https://user-images.githubusercontent.com/51548107/190868830-1924c5d4-96fa-440c-9001-b11a4dd81f9d.png">

Tip eloston-chromium is the most power efficient browser by far, better even than safari.

**Comment by bachig26** [comment]
> Tip eloston-chromium is the most power efficient browser by far, better even than safari.

@33ddy have you tried [Orion Browser](https://browser.kagi.com)? which is better and efficient in my experience. Available in App Store, and .dmg also for mac.

**Reply by 33ddy** [reply]
I tried it ser for 3 months, l think you didn’t try eloston-chromium do it ser and thanks me later.
It’s pain to setup it in the beginning,I use Raindrop.io for bookmarks.
You will be amazed with battery.
